tests/regression/test_suite/send_transfer.py

```diff
@@ -120,7 +192,18 @@ def setUpClass(cls):
                             [0, None, rand_tag, rand_addr],
                             [0, rand_msg, None, rand_addr],
                             [0, rand_msg, rand_tag, None],
-                            [0, rand_msg, rand_tag, "我思故我在"]]
+                            [0, rand_msg, rand_tag, "我思故我在"],
+                            [0, rand_msg, "*CCr-m<2FSv9]7=\"mK=LW", rand_addr],
```

Comment:
Don't do that. You can improve the randomized messages in other pull request(s).